### PR TITLE
chore: docusaurus build on Windows with GitHub actions

### PR DESCRIPTION
**Pull request by endiliey**
## Motivation

Testing GitHub actions to build Windows for both v1 and v2 to prevent regression

all code is kinda self-made from reading https://help.github.com/en/actions/automating-your-workflow-with-github-actions

Seems to work well

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

TBD. CI should appear ??
image

image

image

## Gonna release 2.0.0-alpha.39 soon for hotfix

### CODE REVIEW

**Comment by endiliey** [comment]
@JoelMarcey is it okay to use GitHub actions ? Just making sure

File: .github/workflows/nodejs-windows.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        node-version: [8.x]
```
Comment (yangshun) [inline .github/workflows/nodejs-windows.yml]: Since we have a minimum required Node version, is it possible to use that here?

Comment (endiliey) [reply .github/workflows/nodejs-windows.yml]: You have to pin exact version. Or just use .x which means the latest stable version of that major node version. Our min is 8.10.0 anyway which is the earlier version of 8.

Comment (endiliey) [reply .github/workflows/nodejs-windows.yml]: Well this is the same as our circleci config. 8.x

**Comment by endiliey** [comment]
Hmm relay’s one is on linux. We have that config on our circleci. I explicitly use gh actions since it supports windows build. 
For normal build (on linux), we rely on netlify

I think its not a good idea yet to move to gh action since its still new and I notice slower build on gh actions than circleci. Im also more familiar with circleci and/or travis

Maybe worth a revisit in future

**Comment by endiliey** [comment]
Its also hard for me to configure the move because we need to set the environment variable (such as crowdin api key). I also have limited access to this repo, example: “Setting”.

**Comment by endiliey** [comment]
Mentioning that, I also dont have access to docusaurus crowdin api key 🤣 so I cant help on the move. The env is all hidden in circleci

**Comment by endiliey** [comment]
Uhh .. github mobile beta didn't squash and merge the PR 😢

**Comment by JoelMarcey** [comment]
@endiliey I think we can get you access to the api key. Would that help you immediately?

**Comment by endiliey** [comment]
@joelmarcey I think we dont need to do anything for now. The thing is, GitHub action requires full access to the repo to setup environment variable (in particular the “setting”). But I think that can be very dangerous. Lets stick with circleci for now

**Pull request merged**